mailpoet/lib/EmailEditor/Engine/StylesController.php

```diff
@@ -5,9 +5,9 @@
 class StylesController {
   /**
    * Width of the email in pixels.
-   * @var int
+   * @var string
    */
-  const EMAIL_WIDTH = 660;
+  const EMAIL_WIDTH = '660px';
```

Comment:
Could you please explain a bit why we need this change? To me, it seems that this forces us to use `parseNumberFromStringWithPixels` to get back the int value. Maybe we could return values with units only from the `BlocksWidthPreprocessor` and keep int?

Reply:
When we work with the email width in the preprocessor, it was for me more clear to work with a value containing units. Also, we build email layout styles in `SettingsController`, and there should not be width without units.
Of course, it's not necessary and I can change it or we can change it later when it is too annoying for us.

Reply:
Ok. Let's continue with the units in value. I guess we will see soon enough what's working better for us.